grouping.json

```diff
@@ -0,0 +1,153 @@
+    "disallowFunctionDeclarations"
+  ],
+  "Keywords": [
+    "disallowPaddingNewlinesBeforeKeywords",
+    "requirePaddingNewlinesBeforeKeywords",
+    "requireKeywordsOnNewLine",
+    "disallowKeywordsOnNewLine",
+    "disallowKeywords"
+  ],
+  "Variables & Naming": [
+    "disallowMultipleVarDecl",
+    "requireMultipleVarDecl",
+    "requireLineBreakAfterVariableAssignment",
+    "requirePaddingNewLineAfterVariableDeclaration",
+    "requireCamelCaseOrUpperCaseIdentifiers",
+    "requireCapitalizedConstructors",
+    "disallowIdentifierNames",
+    "disallowImplicitTypeConversion",
+    "requireMatchingFunctionName",
+    "requireNamedUnassignedFunctions"
+  ],
+  "Objects Keys": [
+    "requirePaddingNewLinesInObjects",
+    "disallowPaddingNewLinesInObjects",
+    "disallowQuotedKeysInObjects",
+    "requireDotNotation",
+    "disallowSpaceAfterObjectKeys",
+    "requireSpaceBeforeObjectValues",
+    "requireSpaceAfterObjectKeys",
+    "disallowSpaceBeforeObjectValues",
+    "disallowObjectKeysOnNewLine",
+    "requireObjectKeysOnNewLine",
+    "requireAlignedObjectValues"
+  ],
+  "Curly Braces & Brackets": [
+    "requireCurlyBraces",
+    "requireSpacesInsideObjectBrackets",
+    "disallowSpacesInsideArrayBrackets",
+    "disallowSpacesInsideParentheses",
+    "disallowSpacesInsideObjectBrackets",
+    "requireSpacesInsideArrayBrackets",
+    "requireSpacesInsideParentheses",
+    "disallowSpacesInForStatement",
+    "disallowSpacesInsideBrackets",
+    "requireSpacesInForStatement",
+    "requireSpacesInsideBrackets"
+  ],
+  "Unary & Binary operators": [
+    "disallowOperatorBeforeLineBreak",
+    "requireOperatorBeforeLineBreak",
+    "disallowSpaceAfterPrefixUnaryOperators",
+    "disallowSpaceBeforePostfixUnaryOperators",
+    "requireSpaceAfterPrefixUnaryOperators",
+    "requireSpaceBeforePostfixUnaryOperators",
+    "requireSpaceBeforeBinaryOperators",
+    "requireSpaceAfterBinaryOperators",
+    "disallowSpaceBeforeBinaryOperators",
+    "disallowSpaceAfterBinaryOperators"
+  ],
+  "Comments": [
+    "requireSpaceAfterLineComment",
+    "requireCapitalizedComments",
+    "disallowSpaceAfterLineComment",
+    "disallowCapitalizedComments",
+    "disallowKeywordsInComments",
+    "disallowPaddingNewLinesBeforeLineComments",
+    "requirePaddingNewLinesBeforeLineComments"
+  ],
+  "Maximums": [
+    "maximumLineLength",
+    "maximumNumberOfLines"
+  ],
+  "ES2015/ES6": [
+    "disallowParenthesesAroundArrowParam",
+    "requireParenthesesAroundArrowParam",
+    "requireArrowFunctions",
+    "requireSpread",
+    "requireTemplateStrings"
+  ],
+  "Everything else": [
+    "requireParenthesesAroundIIFE",
+    "disallowImplicitTypeConversion",
+    "disallowMultipleLineStrings",
+    "validateQuoteMarks",
+    "disallowYodaConditions",
+    "requireYodaConditions",
+    "disallowNodeTypes",
+    "disallowNotOperatorsInConditionals",
+    "requireDollarBeforejQueryAssignment",
+    "requireNumericLiterals",
```

Comment:
Small thing but I added http://jscs.info/rule/requireNumericLiterals.html as a es6 rule since it involves binary/octal literals